### PR TITLE
MAP-29 Add "Create Mapping" page

### PR DESCRIPTION
**Pull request by AdamGrzybkowski**

### CODE REVIEW

File: owa/app/js/home/home.html [hunk 1]
```diff
@@ -1,3 +1,6 @@
+<div>
+    <a class="button" href="#/mapping"><i class="icon-plus"></i>&#160;Create Mapping</a>
```
Comment (dkayiwa) [inline owa/app/js/home/home.html]: Shouldn't we localise the create mappings text?

Comment (AdamGrzybkowski) [inline owa/app/js/home/home.html]: @dkayiwa I will do it once [this PR ](https://github.com/openmrs/openmrs-module-metadatamapping/pull/38/files) is merged. Is that ok ? :)

Comment (dkayiwa) [inline owa/app/js/home/home.html]: @AdamGrzybkowski so are you planning to merge that other pull request? :)

Comment (AdamGrzybkowski) [inline owa/app/js/home/home.html]: @dkayiwa I thought you are waiting for @rkorytkowski opinion, but I can merge it :)

Comment (dkayiwa) [inline owa/app/js/home/home.html]: @AdamGrzybkowski it looks fine to me. If you do not get a response from him, go ahead and merge. :) Am more inclined to merging and making followup fixes than having a pull request open for weeks! :)

**Comment by dkayiwa** [comment]
@AdamGrzybkowski did you notice the merge conflicts?

**Comment by AdamGrzybkowski** [comment]
@dkayiwa @PawelGutkowski @rkorytkowski 
PR updated

**Pull request merged**